### PR TITLE
Update PR template to reflect the deletion of FxCopAnalyzers.sln

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

**Review by Evangelink — APPROVED** [review]
Good catch!

**Comment by Evangelink** [comment]
Relates to #4504

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!--
 
-Make sure to run `msbuild RoslynAnalyzers.sln /t:pack` and `msbuild FxCopAnalyzers.sln /t:pack` in the repository root when
+Make sure to run `msbuild RoslynAnalyzers.sln /t:pack` in the repository root when
```
Comment (mavasani) [inline .github/PULL_REQUEST_TEMPLATE.md:3]:
```suggestion
Make sure to run `msbuild /t:pack` in the repository root when
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <!--
 
-Make sure to run `msbuild RoslynAnalyzers.sln /t:pack` and `msbuild FxCopAnalyzers.sln /t:pack` in the repository root when
+Make sure to run `msbuild /t:pack` in the repository root when
```
Comment (Evangelink) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: I would recommend to add the `/v:m` to have the minimal verbosity which speeds up the process.

Comment (Youssef1313) [reply .github/PULL_REQUEST_TEMPLATE.md:3]:
```suggestion
Make sure to run `msbuild /t:pack /v:m` in the repository root when
```

**Review by Evangelink — APPROVED** [review]

**Review by mavasani — APPROVED** [review]

**Pull request merged**